### PR TITLE
Update core-developers.yml

### PR DESCRIPTION
**Pull request by samwaseda**
Remove people who are not there anymore.

### CODE REVIEW

**Comment by jan-janssen** [comment]
I guess we should add them to the alumni list: https://github.com/pyiron/pyiron.github.io/blob/update-core-developers/_data/alumni.yml

File: _data/alumni.yml [hunk 1]
```diff
@@ -1,4 +1,20 @@
 # Previous pyiron developers who are not on Github
+-
+  name: Muhammad Hassani
+  position: Core Developer
+  image_path: https://www.mpie.de/employee_images/107319-1610705257?t=eyJ3aWR0aCI6NDI0LCJoZWlnaHQiOjU0NSwiZml0IjoiY3JvcCIsImZpbGVfZXh0ZW5zaW9uIjoianBnIn0%3D--cff5e640ea2ce8013d78b56c0ecfd3df99788a11
+  links:
+      professional: https://www.mpie.de/person/107319/3079071
```
Comment (samwaseda) [inline _data/alumni.yml:7]:
```suggestion
```

File: _data/alumni.yml [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
+      github: muh-hassani
+  dates: 2021-2022
+-
+  name: Sudarsan Surendralal
+  position: Core Developer
+  image_path: https://www.mpie.de/employee_images/41328-1554806523?t=eyJ3aWR0aCI6NDI0LCJoZWlnaHQiOjU0NSwiZml0IjoiY3JvcCJ9--097e0e16b1239d83449ff2fd629b525b0f1936f9
+  links:
+      professional: https://www.mpie.de/person/41328/2281
```
Comment (samwaseda) [inline _data/alumni.yml:15]:
```suggestion
```

File: _data/alumni.yml [hunk 3]
```diff
@@ -1,4 +1,18 @@
 # Previous pyiron developers who are not on Github
+-
+  name: Muhammad Hassani
+  position: Core Developer
+  image_path: https://www.mpie.de/employee_images/107319-1610705257?t=eyJ3aWR0aCI6NDI0LCJoZWlnaHQiOjU0NSwiZml0IjoiY3JvcCIsImZpbGVfZXh0ZW5zaW9uIjoianBnIn0%3D--cff5e640ea2ce8013d78b56c0ecfd3df99788a11
```
Comment (samwaseda) [inline _data/alumni.yml]:
```suggestion
  contribution: Development of the infrastructure
```

File: _data/alumni.yml [hunk 4]
```diff
@@ -1,4 +1,18 @@
...
+  links:
+      github: muh-hassani
+  dates: 2021-2022
+-
+  name: Sudarsan Surendralal
+  position: Core Developer
+  image_path: https://www.mpie.de/employee_images/41328-1554806523?t=eyJ3aWR0aCI6NDI0LCJoZWlnaHQiOjU0NSwiZml0IjoiY3JvcCJ9--097e0e16b1239d83449ff2fd629b525b0f1936f9
```
Comment (samwaseda) [inline _data/alumni.yml]:
```suggestion
  contribution: Development of core functionality and maintenance of VASP
```

**Pull request merged**